websites/S/Stats.fm/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "2Lost4Discord",
+		"id": "421991668556759042"
+	},
+	"contributors": [
+		{
+			"name": "BigBoi©",
+			"id": "853932304257908746"
+		}
+	],
+	"service": "Stats.fm",
+	"description": {
+		"en": "Enter a new dimension of music by getting unique insights into your music taste."
+	},
+	"url": "stats.fm",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/WkUe5aL.png",
+	"thumbnail": "https://i.imgur.com/WkUe5aL.png",

```

Comment:
the thumbnail can't be the logo

Reply:
Currently i don't have a banner, what should i do? Can i leave it without for now?

Reply:
Take a screenshot of the website's main page

Reply:
> Take a screenshot of the website's main page

Alright, thanks, im also adding more supported pages currently